x-pack/plugins/apm/public/components/app/correlations/use_correlations_search_strategy.ts

```diff
@@ -0,0 +1,109 @@
+/*
```

Comment:
Can we rename this to `use_failure_correlations.ts` for consistency? The other related file there is `use_correlations.ts` (which we probably should rename to `use_latency_correlations.ts` but suggest to not do this in this PR)

Reply:
Updated here https://github.com/elastic/kibana/pull/108441/commits/155443605ee491b2eb7c2bcb7b3e458eed9591e6

Reply:
I think in the future, as a follow up in 7.16, we should consolidate the hook for the two search strategies since they share a lot of similar code.